regex-syntax/src/parser.rs

```diff
@@ -1047,19 +1058,19 @@ fn rev_concat(mut exprs: Vec<Expr>) -> Expr {
     }
 }
 
-// Returns ture iff the given character is allowed in a capture name.
+// Returns true if the given character is allowed in a capture name.
```

Comment:
Oh, haven't seen that before. (Haven't read any math papers in English, though.) I'll change it to "if and only if".